internal/output/prefixed.go

```diff
@@ -85,11 +86,11 @@ func (pw *prefixWriter) writeLine(line string) error {
 		line += "\n"
 	}
 
-	idx, ok := pw.prefixed.seen[pw.prefix]
+	idx, ok := pw.prefixed.seen.Load(pw.prefix)
```

Comment:
Perhaps LoadOrStore() is better than Load() and then Store()?

Reply:
You're right, that would be neater. But I will await the below discussion about `sync.Map`.